app/brave_main_delegate.cc

```diff
@@ -46,7 +46,13 @@ BraveMainDelegate::CreateContentBrowserClient() {
 #if defined(CHROME_MULTIPLE_DLL_CHILD)
   return NULL;
 #else
-  return g_brave_content_browser_client.Pointer();
+  if (chrome_content_browser_client_ == nullptr) {
+    DCHECK(service_manifest_data_pack_);
+    chrome_content_browser_client_ =
+        std::make_unique<BraveContentBrowserClient>(
+            std::move(service_manifest_data_pack_));
+  }
+  return chrome_content_browser_client_.get();
```

Comment:
This updates to what chrome main delegate does. In particular it was a crash because it has first argument with a default value of nullptr, and we didn't use that first argument until now.